sdk/master_board_sdk/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package>
+    <name>master_board_sdk</name>
+    <version>1.0.0</version>
+    <description>
+        This project contains the sdk for the communication between a computer
+        and the master-board
+    </description>
+    <maintainer email="mnaveau@tue.mpg.de">Maximilien Naveau</maintainer>
+    <license>BSD 3-Clause</license>
```

Comment:
I guess that's `BSD-3-Clause`

Reply:
I have always seen it written `BSD 3-Clause`
https://tldrlegal.com/license/bsd-3-clause-license-(revised)
https://github.com/machines-in-motion/ubuntu_installation_scripts/blob/master/LICENSE
https://opensource.org/licenses/BSD-3-Clause (3-clause BSD ...)

Anyway I do not have strong opinion again...

Reply:
Ok, I thought that http://wiki.ros.org/catkin/package.xml would recommend using the spdx-id, but it looks like you can put pretty much anything you want, thanks

Reply:
I did not know about this spdx-id. My bad. But indeed this field should be able to contain anything.
I think it is more for human to read than for some code parsing. So maybe no need for a norm here?
I do not know.